android/app/build.gradle

```diff
@@ -386,9 +386,6 @@ dependencies {
         implementation jscFlavor
     }
 
-    implementation 'androidx.appcompat:appcompat:1.1.0-rc01'
-    implementation 'androidx.swiperefreshlayout:swiperefreshlayout:1.1.0-alpha02'
```

Comment:
these are no longer needed since `react-native-screens` specifies them directly now

https://github.com/software-mansion/react-native-screens/blob/6c87d7749ec62fbb51fb4ec50af1fa8733ebae86/android/build.gradle#L60-L61